log/scope.go

```diff
@@ -315,3 +347,103 @@ func (s *Scope) SetLogCallers(logCallers bool) {
 func (s *Scope) GetLogCallers() bool {
 	return s.logCallers.Load().(bool)
 }
+
+// Copy makes a copy of s and returns a pointer to it.
+func (s *Scope) Copy() *Scope {
```

Comment:
Is there a use case for this or should it be private?

Reply:
yes, thanks for pointing out, it should be now.